src/systemctl/systemctl-list-units.c

```diff
@@ -321,7 +321,7 @@ static int socket_info_compare(const SocketInfo *a, const SocketInfo *b) {
         if (r != 0)
                 return r;
 
-        r = strcmp(a->path, b->path);
+        r = path_compare(a->path, b->path);
```

Comment:
i.e., maybe:

```c
r = CMP(a->path[0] == '/', b->path[0] == '/');
if (r != 0)
        return r;
r = a->path[0] == '/' ? path_compare(a->path, b->path) : strcmp(a->path, b->path);
```

or so?

Reply:
Thanks. Applied. Upgrading the green label.